book/MCDM Productions; Arcadia Issue 8.json

```
@@ -0,0 +1,3593 @@
+                            "Sea"
+                        ],
+                        [
+                            "4",
+                            "Coven"
+                        ]
+                    ]
+                }
+            ]
+        },
+        {
+            "name": "Expanded Spell List",
+            "source": "Ar8",
+            "className": "Warlock",
+            "classSource": "PHB",
+            "subclassShortName": "Hag",
+            "subclassSource": "Ar8",
+            "page": 5,
+            "level": 1,
+            "entries": [
+                "The Hag lets you choose from an expanded list of spells when you learn a warlock spell. The following spells are added to the warlock spell list for you, determined by your patron’s kind: green, night, sea, or coven.",
+                {
+                    "type": "table",
+                    "colLabels": [
+                        "Spell Level",
+                        "Green Hag Spells",
+                        "Night Hag Spells",
+                        "Sea Hag Spells",
+                        "Coven Spells"
+                    ],
+                    "colStyles": [
+                        "col-2 text-center",
+                        "col-2 text-center",
+                        "col-2 text-center",
+                        "col-2 text-center",
+                        "col-2 text-center"
+                    ],
+                    "rows": [
+                        [
+                            "1st",
+                            "{@spell detect magic}, {@spell silent image}",
+                            "{@spell magic missile}, {@spell sleep}",
+                            "{@spell create or destroy water}, {@spell fog cloud}",
+                            "{@spell animal friendship}, {@spell tasha's hideous laughter|hideous laughter}"
+                        ],
+                        [
+                            "2nd",
+                            "{@spell detect thoughts}, {@spell spike growth}",
+                            "{@spell alter self}, {@spell blindness/deafness}",
+                            "{@spell moonbeam}, {@spell silence}",
+                            "{@spell locate object}, {@spell web}"
+                        ],
+                        [
+                            "3rd",
+                            "{@spell  bestow curse}, {@spell conjure animals}",
+                            "{@spell blink}, {@spell slow}",
+                            "{@spell water walk}, {@spell wind wall}",
+                            "{@spell lightning bolt}, {@spell leomund's tiny hut}"
+                        ],
+                        [
+                            "4th",
+                            "{@spell dominate beast}, {@spell polymorph}",
+                            "{@spell faithful hound}, {@spell phantasmal killer}",
+                            "{@spell evard's black tentacles|black tentacles}, {@spell control water}",
```

Same thing as before.
```suggestion
                            "{@spell evard's black tentacles||black tentacles}, {@spell control water}",
```